docs/userguide/parameters.md

```diff
@@ -1,4 +1,205 @@
 # Parameters
 
-This page will soon contain information about parameters.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+Parameters are fundamental for BayBE, as they configure the ```SearchSpace``` and serve 
+as the direct link to the controllable variables in your experiment. 
+Before starting an iterative campaign, the user is required to specify the exact 
+parameters they can control and want to consider in their optimization.
+
+```{note}
+BayBE identifies each parameter by a ```name```. All parameter names in one 
+campaign must be unique.
+```
+
+BayBE distinguishes several parameter types. Because under the hood they need to be 
```

Comment:
Shouldn't this be "Because they need .... under the hood"? Not sure about the grammar though, just my feeling.